staging/src/k8s.io/apimachinery/pkg/util/httpstream/spdy/roundtripper.go

```diff
@@ -299,7 +299,7 @@ func (s *SpdyRoundTripper) NewConnection(resp *http.Response) (httpstream.Connec
 		} else {
 			// TODO: I don't belong here, I should be abstracted from this class
 			if obj, _, err := statusCodecs.UniversalDecoder().Decode(responseErrorBytes, nil, &metav1.Status{}); err == nil {
-				if status, ok := obj.(*metav1.Status); ok {
+				if status, ok := obj.(*metav1.Status); ok && *status != (metav1.Status{}) {
```

Comment:
Can you add a comment (basically what's in your PR description) for why you're comparing against a zero value Status?

Reply:
I think you're trying to compare against nil of type metav1.Status?  You can do that with `((*metav1.Status)(nil))`

Reply:
If status were nil this would panic, wouldn't it?